certified-connectors/Contacts Pro/apiDefinition.swagger.json

```diff
@@ -0,0 +1,595 @@
+              "$ref": "#/definitions/Contact"
+            }
+          },
+          "400": {
+            "description": "The provided id is invalid"
+          },
+          "404": {
+            "description": "The contact has not been found or does not exists"
+          }
+        },
+        "operationId": "GetContact"
+      },
+      "delete": {
+        "tags": [
+          "Contacts"
+        ],
+        "summary": "Delete a contact",
+        "description": "Delete a contact",
+        "parameters": [
+          {
+            "in": "path",
+            "name": "contactListId",
+            "description": "The contact list unique identifier",
+            "required": true,
+            "type": "string",
+            "format": "uuid",
+            "x-ms-summary": "Contact list ID",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "in": "path",
+            "name": "contactId",
+            "description": "The contact unique identifier",
+            "required": true,
+            "type": "string",
+            "format": "uuid",
+            "x-ms-summary": "Contact ID",
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Return no content"
+          },
+          "400": {
+            "description": "The provided id is invalid"
+          },
+          "404": {
+            "description": "The contact has not been found or does not exists"
+          }
+        },
+        "operationId": "DeleteContact"
+      },
+      "patch": {
+        "tags": [
+          "Contacts"
+        ],
+        "summary": "Update a contact",
+        "description": "Update a contact",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "contactListId",
+            "description": "The contact list unique identifier",
+            "required": true,
+            "type": "string",
+            "format": "uuid",
+            "x-ms-summary": "Contact list ID",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "in": "path",
+            "name": "contactId",
+            "description": "The contact unique identifier",
+            "required": true,
+            "type": "string",
+            "format": "uuid",
+            "x-ms-summary": "Contact ID",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Contact to update",
+            "required": true,
+            "schema": {
+              "required": [
+                "name"
+              ],
+              "type": "object",
+              "properties": {
+                "name": {
+                  "description": "The name or fullname (firstname and or lastname) of the contact",
```

Comment:
All descriptions, titles, summaries, etc. must be free from grammar and spelling errors. Please review them.